src/controllers/question.js

```diff
@@ -1,12 +1,102 @@
+const Question = require('../models/question');
+
+async function saveQuestion(req) {
+  let { question } = req;
+  question.title = req.body?.title;
+  question.content = req.body?.content;
+  question.subjects = req.body?.subjects;
+  question.isSolved = req.body?.isSolved;
```

Comment:
In case of addNewQuestion, I think there is no need to that lines because the necessary validations are handled by question model( required: true,)